modules/402-ingress-nginx/crds/ingress-nginx.yaml

```diff
@@ -15,1241 +15,1288 @@ spec:
     kind: IngressNginxController
   preserveUnknownFields: false
   versions:
```

Comment:
Why so significant changes?
If you don't like the indents, make a special PR for reformatting.